hash/msx2_cart.xml

```diff
@@ -1904,6 +1916,35 @@ LZ93A13 (32 pin) - 8KB banks
 		</part>
 	</software>
 
+	<software name="arknoid2k" cloneof="arknoid2">
+		<description>Arkanoid 2 (Korea)</description>
+		<year>1988</year>
+		<publisher>Zemina</publisher>
+		<notes>Does not work with Arkanoid controller.</notes>
+		<info name="alt_title" value="알카노이드 II"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="konami"/>
+			<dataarea name="rom" size="0x20000">
+				<rom name="arkanoid 2 - zemina.rom" size="0x20000" crc="1a7bdaa8" sha1="c843b8b0e5ed98d3bae68e701fb709ea178ed968" status="baddump"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="ashguin3b" cloneof="ashguin3">
+		<description>Ashguine - Fukushuu no Honoo (Japan, alt 2)</description>
+		<year>1987</year>
```

Comment:
Should be “AshGuine - Fukushuu no Honoo” (capitalisation).  Parent as well?

Reply:
Fixed. The parent and a few other sets also had "Ashguine".